Server/Main.cs

```diff
@@ -523,7 +541,7 @@ public static void Main(string[] args)
 
 				Console.WriteLine(" - Press return to exit, or R to try again.");
 
-				if (Console.ReadKey(true).Key != ConsoleKey.R)
+				if (Insensitive.Equals(Console.ReadLine(), "R"))

```

Comment:
IMHO no need to change this, there is a check for `Core.Service` two lines above. Just change it for `Core.NoConsole` should be enough.